### DISCUSSION TITLE
[Need Help] Can't pin tiles using start menu WIND-10

### DISCUSSION THREAD

**Discussion by Larsmeneer** [valinet/ExplorerPatcher#1330] [General]
Hello!

I have just made the switch because I viewed your program, 
but when I try to pin tiles to the start menu (classic windows 10) (windows 11 operating sys) it doesnt show up sadly, can you help me?

Kind regards,
Lars

**Comment by Shengfuih** [comment]
Same here. I got some minor issues and resetted EP. Now some tiles in W10 startmenu are missing (weather, calendar, spotify, whatsapp and so on), i cannot add these apps to start. Next Problem is, these apps dont show up in the programs list, so i have to search for the every time.

**Reply by Shengfuih** [reply]
Followed the instructions to set EP to use "Win11 Startmenu -> restart EP -> Use Win10 Startmenu -> restart EP -> and then click at EP-Settings "Start Menu" "Pin tiles to Windows 10 Start menu from File Explorer". Sometimes you have to click Add to startmenu up to 5 times to make the tile appear, but now it worked.

The missing tiles after reset came from this location c:\Program Files\WindowsApps\c:\Program Files\WindowsApps\
you can only access this folder under admin rights, All of the Apps from this folder where missing. Before the reset, the tiels of weather, calendar and mail were interactive, now i have only the icon.

**Comment by Larsmeneer** [comment]
Thanks!
Now I can use my Windows 11 installation with the features that I dont like about it :)
